weasyl/oauth2.py

```diff
@@ -66,21 +74,15 @@ def authorize_post_(request):
                                     form.username, form.password, bool(form.remember_me),
                                     bool(form.not_me)))
     credentials['userid'] = userid
-    headers, body, status = server.create_authorization_response(
+    response = server.create_authorization_response(
```

Comment:
I'd prefer to reserve the name `response` for Pyramid response objects only. Everywhere else in the code-base that's what it refers to, so avoiding breaking that here was a part of least surprise.